e2e/global/common/kamelet_binding_test.go

```diff
@@ -143,3 +172,29 @@ func createErrorProducerKamelet(ns string, name string) func() error {
 
 	return CreateKamelet(ns, name, flow, props, nil)
 }
+
+func createTimerKamelet(ns string, name string) func() error {
```

Comment:
There is already `CreateTimerKamelet` in test_support.go. Should we still deplicate the func here?